vassal-app/src/main/java/VASSAL/build/module/map/DeckGlobalKeyCommand.java

```diff
@@ -137,13 +150,18 @@ public void decode(String s) {
     setAttribute(DECK_COUNT, sd.nextInt(0));
     setAttribute(REPORT_FORMAT, sd.nextToken(""));
     localizedName = sd.nextToken(getConfigureName());
+    setAttribute(TARGET, sd.nextToken(""));
+
   }
 
   @Override
   public String[] getAttributeDescriptions() {
     return new String[]{
       Resources.getString(Resources.NAME_LABEL),
       Resources.getString("Editor.GlobalKeyCommand.command"), //$NON-NLS-1$
+
+      Resources.getString("Editor.GlobalKeyCommand.pre_select"), // Fast Match parameters
+
```

Comment:
I though we where removing Fast Match from DeckGlobalKeyCommand? If we do, the 'Additional Match Properties' label will need to revert to 'Matching properties'.

Reply:
good point